crates/loro-internal/src/delta/tree.rs

```diff
@@ -31,6 +32,7 @@ pub enum TreeExternalDiff {
         position: FractionalIndex,
     },
     Delete,
+    EmptyTrash,
```

Comment:
Should we show user which nodes have been emptied?

Reply:
Can it be treated as Delete as well? In user land, we don't have to differentiate the two, should we?

Reply:
I don't think so. For users, `Delete` means delete the node, and `EmptyTrash` means **delete (clean)** the deleted node. We can't treat them the same.

Currently I only emit an event to tell the user that the `Trash` has been emptied.

Reply:
But this is just an event. What are the differences from the user's perspective? It's confusing if users should handle them in the same way.